Source/tusdotnet/Adapters/RequestAdapter.cs

```diff
@@ -16,6 +16,10 @@ internal sealed class RequestAdapter
 
         public Stream Body { get; set; }
 
+#if NETCOREAPP3_0
```

Comment:
I think it would be better to define "features" in the build target, e.g. "pipereader" and use that as compile directives. This way we could add new build targets (e.g. net5) without having to update the code. It would also work with the endpoint routing suggestion from #116.